src/Raven.Client/Documents/Operations/Expiration/ExpirationConfiguration.cs

```diff
@@ -8,11 +8,16 @@ public class ExpirationConfiguration : IDynamicJson
 
         public long? DeleteFrequencyInSec { get; set; }
 
+        public long? ExpirationMaxItemsToProcess { get; set; }
```

Comment:
remove 'Expiration' ?

Reply:
I thought it would be confusing but sure, done